pkg/sfu/buffer/rtpstats.go

```diff
@@ -79,7 +79,6 @@ type Snapshot struct {
 }
 
 type SnInfo struct {
-	pktTime       int64
```

Comment:
staticcheck, I had missed removing this field in the PR where I reverted using per packet time.